### DISCUSSION TITLE
Last-minute switch to backup crew

### DISCUSSION THREAD

**Discussion by peyre** [raceintospace/raceintospace#599] [Ideas]
I've wondered if it might be a good idea to have a slight chance that shortly before launch, something goes wrong with the primary crew and the game switches the mission to the backup crew (if any).  Somewhat analogous to how Mattingly was pulled from the Apollo 13 crew and replaced by his counterpart on the backup crew because he'd been exposed to rubella.  The player might get a message right before last-minute confirmation (below).  This would be an additional advantage to assigning backup crews when you can.

![Backup Crew](https://user-images.githubusercontent.com/3118014/140332932-de5f1f87-c462-4cb0-9e58-26df7168cb0d.png)
![Scheduled launch](https://user-images.githubusercontent.com/3118014/140332976-7a8a9e0c-5b5b-4773-be31-43698060b7c2.png)

**Comment by hweimer** [comment]
Isn't there a newscast for this (IIRC involving the common cold instead of measles)?

**Reply by peyre** [reply]
There is, but it's at the start of your turn.  I got to thinking, what if there was also a (very low) chance of a switchout being forced _after_ your turn?

I'm starting to think this wasn't such a great idea though.

**Reply by hweimer** [reply]
You are right that it's a bit weird that the game throws a newscast at you about an event lasting  for a few days, at a time when it's not even clear during which *month* the launch will take place. But I guess that's just how the game works, the storm blocking one of the pads is a similar situation.

**Reply by peyre** [reply]
I think the origin of that is that in _Liftoff!_ you draw random event cards, which you'd assume meant something which will happen at some point during your turn.  Whereas in (BA)RIS the random events are dressed up as a newscast, which was a great idea, but could lead to some oddities as you mention.
